### PR TITLE
Update TaskFailureInformationCodes in BatchErrorCodeStrings.cs

### PR DESCRIPTION
**Pull request by manelone**
Update TaskFailureInformationCodes with codes for resource container access denied on resource downloads and file uploads

### CODE REVIEW

File: sdk/batch/Microsoft.Azure.Batch/src/Common/BatchErrorCodeStrings.cs [hunk 1]
```diff
@@ -710,6 +710,16 @@ public static class TaskFailureInformationCodes
         /// </summary>
         public const string DiskFull = "DiskFull";
 
+        /// <summary>
+        /// Access was denied when trying to upload a file to an output destination specified for the task.
+        /// </summary>
+        public const string FileUploadAccessDenied = "FileUploadAccessDenied";
```
Comment (AlexGhiondea) [inline sdk/batch/Microsoft.Azure.Batch/src/Common/BatchErrorCodeStrings.cs]: This requires an updated package version.

**Comment by AlexGhiondea** [comment]
@matthchr could you take a look?

**Comment by matthchr** [comment]
These constants are already defined in that same file down below in the `TaskFailureInformationCodes` class.

These codes are the cause of "task failures" and so have their own category.

I don't think these codes should be on the `BatchErrorCodeStrings` class as `BatchErrorCodeStrings` are specifically codes the service returns as part of `BatchError` when a REST API call hit an error.

**Comment by AlexGhiondea** [comment]
@manelone could you please take a look at the feedback on this PR?

**Comment by jsquire** [comment]
Thank you, @manelone, for your contribution.

This request hasn't move forward in some time, and seems to have stalled.   I'm going to close this out as stale; please feel free to reopen if you would like to continue working on these changes.

**Pull request closed without merging**